src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.shared.cs

```diff
@@ -20,7 +20,9 @@ public Snackbar()
 		VisualOptions = new SnackbarOptions();
 	}
 
-	/// <inheritdoc/>
+	/// <summary>
+	/// Returns true if the <see cref="Snackbar"/> is currently visible.
```

Comment:
Details, but `true` could also be `<see langword="true"/>` to make it show up with a bit of syntax highlighting in IntelliSense, same for all keywords.

Reply:
```suggestion
	/// Returns <see langword="true"/> if the <see cref="Snackbar"/> is currently visible.
```